api/server.go

```diff
@@ -34,6 +34,10 @@ func NewServer(config Config) *echo.Echo {
 	e.POST("/agreements", PostAgreementsHandler(config.DB))
 	e.PUT("/documents/:name", PutDocumentHandler(config.DB))
 	e.GET("/documents/:name", GetDocumentHandler(config.DB))
+	e.GET("/users/:uuid", GetUserHandler(config.DB))
+	e.GET("/users", GetUsersHandler(config.DB))
+	e.PUT("/users/:uuid", UserHandler(config.DB))
```

Comment:
The name of `UserHandler` with PUT and PATCH should match the convention established by other handlers